README.md

```diff
@@ -1,3 +1,11 @@
+<p align="center">
+  <a href="http://www.scala-js.org/">
+    <img src="http://www.scala-js.org/assets/img/scala-js-logo.svg" height="128">
+    <h1 align="center">Scala.js</h1> 
+  </a>
+</p>
+
+
```

Comment:
Avoid double blank lines. Use only one blank line here.

Reply:
Thanks for the suggestion! I will update it 👌🏽